src/main/java/bit/javaoop/managersPackage/Manager.java

```diff
@@ -0,0 +1,33 @@
+package bit.javaoop.managersPackage;
+
+import bit.javaoop.employeesPackage.AbstractEmployee;
+import bit.javaoop.employeesPackage.Employee;
+
+import java.util.ArrayList;
+
+
+public abstract class Manager extends AbstractEmployee {
+    protected double budget;
+    protected double employeesLimit;
+    private double budgetLeft = budget;
+
+    public Manager(String name, double salary, double budget, double employeesLimit) {
+        super(name, salary);
+        this.budget = budget;
+        this.employeesLimit = employeesLimit;
+    }
+
+    public static ArrayList<Employee> employeesList;
```

Comment:
powinno być na samej górze przy deklaracji innych zmiennych + nigdzie nie inicjalizujesz tej listy w ten sposób jak teraz stworzyłeś jedynie zmienną, na adres w pamięci Twojej listy ale samej Listy nie musisz jeszcze dodać np. w konstruktorze 
`this.employeesList = new ArrayList<>();`

ciekawostka zmienne prymitywne i takie referencje są tworzone na stosie ( stack ) natomiast słówko `new` alokuje nam już pamięć na faktyczne obiekty na stercie ( heap )